cmd/snap/cmd_pack.go

```diff
@@ -76,7 +76,7 @@ func init() {
 			// TRANSLATORS: This should not start with a lowercase letter.
 			"filename": i18n.G("Output to this filename"),
 			// TRANSLATORS: This should not start with a lowercase letter.
-			"compression": i18n.G("Compression to use (e.g. xz)"),
+			"compression": i18n.G("Compression to use (e.g. xz or lzo)"),
```

Comment:
e.g. is a bit strange given that we actually support only xz and lzo atm but it was there originally

Reply:
I assume some day we will support more, but yes it is a bit strange today